src/main/kotlin/no/nav/amt_enhetsregister/client/BronnoysundClientImpl.kt

```diff
@@ -89,7 +89,7 @@ class BronnoysundClientImpl(
 			.build()
 
 		httpClient.newCall(request).execute().use { response ->
-			if (response.code == HttpStatus.GONE.value()) {
+			if (response.code == HttpStatus.GONE.value() || response.code == HttpStatus.NOT_FOUND.value()) {
```

Comment:
Tror vi må skille mellom GONE og NOT_FOUND.
Hvis ikke vil du få lagt inn en ny 'Fjernet virksomhet' hver gang noen slår opp på et ugyldig orgnr.
`	fun deltaOppdaterUnderenheter() {
		deltaOppdaterEnhet(enhetType = EnhetType.UNDERENHET) {
			val underenhet = bronnoysundClient.hentUnderenhet(it.organisasjonsnummer)

			if (underenhet == null) { // GONE
				log.info("Underenhet orgnr=${it.organisasjonsnummer} er fjernet fra brreg")

				return@deltaOppdaterEnhet EnhetService.UpsertEnhet(
					organisasjonsnummer = it.organisasjonsnummer,
					navn = "Fjernet virksomhet",
					overordnetEnhetOrgNr = UKJENT_VIRKSOMHET_NR
				)
			}`

Reply:
Eller vent. Dette er kun deltaoppdajteringer, så den vil ikke slå inn.

Reply:
Det er tanken min også, at det ikke burde ha noe å si for deltaoppdateringene. Vi burde ikke får noen tilfeldige 404 responser fra de her men jeg vet jo ikke om det er helt sant.